docs/conditions.md

```diff
@@ -6,50 +6,54 @@ weight: 11
 -->
 # Conditions
 
-This document defines `Conditions` and their capabilities.
+- [Overview](#overview)
+- [Configuring a `Condition`](#configuring-a-condition)
+  - [Specifying the condition `check`](#specifying-the-condition-check)
+  - [Specifying `Parameters`](#specifying-parameters)
+  - [Specifying `Resources`](#specifying-resources)
+- [Code examples](#code-examples)
 
-- [Syntax](#syntax)
-  - [Check](#check)
-  - [Parameters](#parameters)
-  - [Resources](#resources)
-- [Labels and Annotations](#labels-and-annotations)
-- [Examples](#examples)
+## Overview
 
-## Syntax
+A `Condition` resource in Tekton allows you to conditionalize the execution of `Tasks` within a `Pipeline`.
+You define each `Condition` within your `PipelineRun` definition and then conditionalize each desired `Task`
```

Comment:
Would be great to include link to `PipelineRun` doc here i.e. 

[PipelineRun](pipelineruns.md)

Reply:
Added link.